assets/src/block-editor/HTMLSidebarHelp/HTMLSidebarHelp.js

```diff
@@ -0,0 +1,9 @@
+import {RawHTML} from '@wordpress/element';
+
+export const HTMLSidebarHelp = props => {
```

Comment:
Not a big deal, but this could be simplified to 
```
export const HTMLSidebarHelp = props => (
    <div className="HTMLSidebarHelp">
        <RawHTML>{props.children}</RawHTML>
    </div>;
);
```